imagebuildah/build.go

```diff
@@ -183,6 +185,7 @@ type Executor struct {
 	squash                         bool
 	labels                         []string
 	annotations                    []string
+	onbuild                        []string
```

Comment:
I commented this out too, along with the call below to fill it up and it doesn't seem to be needed.

Reply:
Removed